src/main/java/picard/fingerprint/HaplotypeProbabilities.java

```diff
@@ -272,4 +274,5 @@ public double getLodMostProbableGenotype() {
             this.v = v;
         }
     }
+    static final int nGeno = Genotype.values().length;
```

Comment:
Is this done because performance was measured and you found that using `values().length` each time was too slow? It seems like in most cases this wouldn't have a measurable benefit.
If you keep it, it should be all caps as it's a static
```suggestion
    static final int NUM_GENOTYPE = Genotype.values().length;
```

Reply:
not performance, just readability and writability